src/Application/Regulation/Command/SaveRegulationLocationCommand.php

```diff
@@ -31,8 +35,11 @@ public static function create(
         Location $location = null,
     ): self {
         $command = new self($regulationOrderRecord, $location);
+        $command->roadType = $location?->getRoadType();
         $command->cityCode = $location?->getCityCode();
         $command->cityLabel = $location?->getCityLabel();
+        $command->administrator = $location?->getAdministrator();
+        $command->roadNumber = $location?->getRoadNumber();
```

Comment:
(Pinaillage) Rester cohérent sur l'ordre des champs ?

```suggestion
        $command->administrator = $location?->getAdministrator();
        $command->roadNumber = $location?->getRoadNumber();
        $command->cityCode = $location?->getCityCode();
        $command->cityLabel = $location?->getCityLabel();
```